source/includes/reference/_borderless.md

```diff
@@ -5,74 +5,140 @@
 > Example Request:
 
 ```shell
-curl -X GET https://api.sandbox.transferwise.tech/v1/borderless-accounts?profileId={profileId} \
+curl -X GET https://api.sandbox.transferwise.tech/v2/borderless-accounts?profileId={profileId} \
      -H "Authorization: Bearer <your api token>" 
 ```
 
 > Example Response:
 
 ```json
 [
-    {
-        "id": 64,
-        "profileId": <your profile id>,
-        "recipientId": 13828530,
-        "creationTime": "2018-03-14T12:31:15.678Z",
-        "modificationTime": "2018-03-19T15:19:42.111Z",
-        "active": true,
-        "eligible": true,
-        "balances": [
-            {
-                "balanceType": "AVAILABLE",
+  {
+    "id": <your borderless account id>,
+    "profileId": <your profile id>,
+    "recipientId": 5742982,
+    "creationTime": "2016-07-01T16:02:29Z",
+    "modificationTime": "2018-10-20T18:37:21.116Z",
+    "active": true,
+    "eligible": true,
+    "balances": [
+      {
+        "currency": "GBP",
+        "amount": {
+          "value": 20.61,
+          "currency": "GBP"
+        },
+        "reservedAmount": {
+          "value": 16.2,
+          "currency": "GBP"
+        },
+        "bankDetails": [
+          {
+            "id": 1000038,
+            "profileId": <your profile id>,
+            "currency": "GBP",
+            "accountHolderName": "Oliver Wilson",
+            "bankCode": {
+              "sortCode": "23-14-70"
+            },
+            "accountNumber": {
+              "default": "11010101"
+            },
+            "address": {
+              "default": {
+                "firstLine": "TransferWise",
+                "secondLine": "56 Shoreditch High Street",
+                "postCode": "E1 6JJ",
+                "city": "London",
+                "country": "United Kingdom",
+                "stateCode": null
+              }
+            },
+            "limits": {
+              "daily": {
+                "currency": "GBP",
+                "value": null
+              },
+              "yearly": {
                 "currency": "GBP",
-                "amount": {
-                    "value": 10999859,
-                    "currency": "GBP"
-                },
-                "reservedAmount": {
-                    "value": 0,
-                    "currency": "GBP"
-                },
-                "bankDetails": null
+                "value": null
+              }
             },
-            {
-                "balanceType": "AVAILABLE",
+            "translations": {
+              "accountLimits.daily": "24 hour receive limit",
+              "accountNumber.default": "Account number",
+              "address.default": "Address",
+              "bankCode.sortCode": "UK Sort Code",
+              "accountLimits.yearly": "Annual receive limit",
+              "accountHolderName": "Account Holder"
```

Comment:
Why is the `translations` inside the response?
I guess I already know the answer but would love to see a better solution to this.
I imagine you forward the translations because we want to be able to control them in our mobile views without releasing new app versions.
If that is the case then we should try to find other solutions as this way we pollute the API with presentation details.